### DISCUSSION TITLE
v3 when?

### DISCUSSION THREAD

**Discussion by Orrison** [awcodes/filament-tiptap-editor#161] [General]
Howdy!

I noticed that v3 for this package is still in beta/pre-release. Is there a particular goalpost we want to hit before moving to v3 as a stable release?

And if so, is there anything I can contribute to help that along?

Thank you!

**Comment by awcodes** [comment]
Thank you. I just haven't had time to get it finalized and fully test due to some pre-existing issues with Livewire 3's entanglement with Alpine.

And I'm also working on a replacement plugin for this. :)

**Reply by lotarbo** [reply]
@awcodes you mean change tiptap to someone else?
Looks like tiptap very useful plugin, and statamic use the same based on prosemirror https://statamic.dev/fieldtypes/bard
I also know https://github.com/portabletext/portabletext which used by sanity https://www.sanity.io/docs/portable-text-editor-configuration

**Reply by Orrison** [reply]
@awcodes got it. No problem at all.

Let me know if you want any assistance in doing so/the replacement. And/or if you would be interested in a better way to handle private media within the plugin, as we discussed in my previous PR.

Thank you for everything!

**Reply by awcodes** [reply]
Not dropping Tiptap. At least not for the foreseeable future, but a better version of the plugin under a different name. Just better to do as a new name than to break the existing packagist stuff.

**Comment by Sebastiangperez** [comment]
Hi , im just trying to follow the instructions on the plugins pages in filamen website but im getting this 
Root composer.json requires awcodes/filament-tiptap-editor ^3.0, found awcodes/filament-tiptap-editor[v3.0.0-alpha1, ..., 3.x-dev] but it does not match your minimum-stability.
Iis this related to the version of filament ? i'm using filament 3 

Also if im do not specify the version im getting two errors 

 Problem 1
    - Root composer.json requires awcodes/filament-tiptap-editor ^2.6 -> satisfiable by awcodes/filament-tiptap-editor[v2.6.0, ..., v2.6.7].
    - awcodes/filament-tiptap-editor[v2.6.0, ..., v2.6.7] require filament/filament ^2.0 -> found filament/filament[v2.5.3, ..., v2.17.53] but it conflicts with your root composer.json require (^3.0-stable).

Thanks

**Reply by awcodes** [reply]
V3 is still in beta follow the install instructions in the readme on the 3.x branch.

**Reply by Sebastiangperez** [reply]
Thanks for your response 

I followed the instructions but  im having the same first error. maybe this is wrong :

composer require awcodes/filament-tiptap-editor:"^3.0"

- Root composer.json requires awcodes/filament-tiptap-editor ^3.0, found awcodes/filament-tiptap-editor[v3.0.0-alpha1, ..., 3.x-dev] but it does not match your minimum-stability.

**Reply by awcodes** [reply]
What is your minimum stability in composer.json. It needs to be "dev" if you're trying to install beta software.

**Comment by Sebastiangperez** [comment]
Yes , Stable , i just change it and works , so sorry man.
